core/templates/core/comments_tree.html

```diff
@@ -0,0 +1,36 @@
+{% load crispy_forms_tags %}
+
+<ul class="list-unstyled">
+    {% for comment in comments %}
+        <li>
+            <div class="card bg-dark shadow-sm mb-3 text-light">
+                {% include 'core/comment.html' with comment=comment %}
+            </div>
+            <div class="mb-3">
+                <button class="btn btn-sm btn-secondary" onclick="toggleForm('commentForm{{ comment.id }}')">Reply</button>
+            </div>
+            <div id="commentForm{{ comment.id }}" class="hidden mt-2 ms-5">
+                <div class="card bg-dark text-light p-3 mb-3">
+                    <form method="post" action="">
+                        {% csrf_token %}
+                        {% if form.non_field_errors %}
+                            <div class="alert alert-danger" role="alert">
+                                {% for err in form.non_field_errors %}
+                                    <p class="form-error">{{ err }}</p>
+                                {% endfor %}
+                            </div>
+                        {% endif %}
+                        {{ form|crispy }}
+                        <input type="hidden" name="parent_id" value="{{ comment.id }}">
```

Comment:
Please add it to the form.  
```
from django import forms
from .models import Post

class PostForm(forms.ModelForm):
    extra_field = forms.CharField(max_length=100, required=False)

    class Meta:
        model = Post
        fields = ['title', 'content']  # Specyfikacja pól modelu

```

Reply:
i dont see how i can not use <input type="hidden" name="parent_id" value="{{ comment.id }}"> in template

Reply:
```
from django.shortcuts import render
from .forms import MyForm

def my_view(request):
    initial_data = {
        'name': 'Jan Kowalski',
        'email': 'example@example.com'
    }
    form = MyForm(initial=initial_data)
    return render(request, 'my_template.html', {'form': form})

```